java/vector/src/main/java/org/apache/arrow/vector/ipc/JsonFileReader.java

```diff
@@ -69,13 +69,23 @@
 
 import io.netty.buffer.ArrowBuf;
 
+/**
+ * A reader for integration test JSON files that converts translates them into Vectors.
+ * This class uses a streaming parser API, method naming tends to reflect this implementation
+ * detail.
```

Comment:
How about "A reader for JSON files that translates them into vectors. This reader is used for integration tests...."

Reply:
SGTM.